public/app/features/alerting/unified/utils/alertmanager.ts

```diff
@@ -243,10 +243,12 @@ export function timeIntervalToString(timeInterval: TimeInterval): string {
   return [timeString, weekdayString, daysString, monthsString, yearsString].join(', ');
 }
 
-export function getTimeString(times?: TimeRange[]): string {
+export function getTimeString(times?: TimeRange[], location?: string): string {
   return (
     'Times: ' +
-    (times ? times?.map(({ start_time, end_time }) => `${start_time} - ${end_time} UTC`).join(' and ') : 'All')
+    (times
```

Comment:
Nay, this is just our presentation of a time string so we can do whatever we want here.

I had initially wanted to refactor the way we render mute timing, making it more visual and showing both the next mute time start and duration but decided it's probably best if I don't make this PR too large.

I'll refactor this function a bit though because it's turning in to quite the monstrosity